lk2nd/device/dts/msm8952/msm8976-qrd.dts

```diff
@@ -9,6 +9,20 @@
 };
 
 &lk2nd {
+	bq-x5-plus {
+		model = "BQ X5 Plus (Longcheer L9360)";
+		compatible = "longcheer,l9360";
+		lk2nd,match-panel;
+
+		panel {
+			compatible = "longcheer,l9360-panel", "lk2nd,panel";
+
+			qcom,mdss_dsi_truly_r63350_1080p_video {
```

Comment:
Downstream dts includes a lot of panels; not sure which are used in production.

Reply:
Our convention so far was:
lk2nd -> add everything as in downstream so we can guarantee device detection (ignoring virtual/default qrd panels of course)
Linux -> only add panels that were tested by someone so we don't upstream dead code later

Reply:
I have added all panels mentioned in downstream.

Is there a convention for proper naming of `compatible` string?

Reply:
There is no explicit convention since we would generate drivers for those ourselves when/if needed (obviously the existing one should match, even if panel selection is not used)